uaclient/tests/test_cli_detach.py

```diff
@@ -99,6 +99,10 @@ def test_entitlements_disabled_appropriately(
                 mock.call(silent=True)
             ] == ent_cls.return_value.can_disable.call_args_list
 
+            assert [
+                mock.call(cfg=cfg, assume_yes=assume_yes)
+            ] == ent_cls.call_args_list
+
```

Comment:
Only because there is already a comment explaining this for loop, we should probably add to it - something like: "# Check that entitlement constructor and can_disable are called correctly"